blocksuite/affine/shared/src/services/telemetry-service/database.ts

```
@@ -18,6 +18,29 @@ export type DatabaseViewEvents = {
+    toViewType: string;
+  };
+};
+
+export type DatabasePropertyEvents = {
+  CreateDatabaseProperty: {
+    propertyType: string; // e.g., 'text', 'number', 'select'
+  };
+};
+
+export type DatabaseFilterEvents = {
+  CreateDatabaseFilter: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)